### PR TITLE
Added first rename spec

### PR DESCRIPTION
**Pull request by kbaum**

### CODE REVIEW

**Review by ynnadkrap — APPROVED** [review]

File: spec/model_renamer/rename_spec.rb [hunk 1]
```diff
@@ -1,3 +1,16 @@
 describe Rename do
-  pending "write it"
+  around do |example|
+    MemFs.activate { example.run }
+  end
```
Comment (ynnadkrap) [inline spec/model_renamer/rename_spec.rb:4]: Thoughts about moving this to `spec_helper`?

```
RSpec.configure do |config|
  config.before do
    MemFs.activate!
  end

  config.after do
    MemFs.deactivate!
  end
end
```

Comment (kbaum) [reply spec/model_renamer/rename_spec.rb:4]: Yes.. i think that will eventually make sense. Thought i would leave it here for now though.

**Comment by pawelgut** [comment]
Thanks for writing tests for this! Never thought I'd live to see this day

**Comment by kbaum** [comment]
This is just a proof of concept! More tests come no this sprint! Danny and
I are going to pair on it!

**Pull request merged**